nbconvert/filters/markdown_mistune.py

```diff
@@ -104,15 +104,20 @@ def header(self, text, level, raw=None):
         html = super(IPythonRenderer, self).header(text, level, raw=raw)
         return add_anchor(html)
 
+    def escape_lt(self,text):
+        return text.replace('<','&lt;')
+
     # Pass math through unaltered - mathjax does the rendering in the browser
```

Comment:
This will no longer be true, we're deciding to follow a different standard than MathJax in this case, as MathJax does not always treat "<" and ">" as characters to be escaped (which leads to the "bugs" that this PR is trying to address).